### PR TITLE
Add capitalize filter and some tests for all default filters

### PR DESCRIPTION
**Pull request by boopathi**

### CODE REVIEW

File: src/test/scala/de/zalando/beard/filter/FilterSpec.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  * @author boopathi
+  */
+class FilterSpec extends FunSpec with Matchers {
+  describe("filters") {
+    it("should capitalize") {
+      val filter = getFilter("capitalize")
+      filter.apply("capitalize") should be("Capitalize")
+      filter.apply("cAPITALIZE") should be("CAPITALIZE")
+    }
+    it("should uppercase") {
```
Comment (danpersa) [inline src/test/scala/de/zalando/beard/filter/FilterSpec.scala:18]: you should leave some empty lines in between tests... and methods...

File: src/test/scala/de/zalando/beard/filter/FilterSpec.scala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      filter.apply("LoWeRcAse") should be("lowercase")
+      filter.apply("LOWERCASE") should be("lowercase")
+    }
+    it("should convert date format") {
+      val filter = getFilter("date")
+      filter.apply("1991-03-10", Map("format" -> "MMddyy")) should be("031091")
+      filter.apply("2011-01-01+01:00", Map("format" -> "D")) should be("1")
+    }
+  }
+  private def getFilter(name: String) =
```
Comment (danpersa) [inline src/test/scala/de/zalando/beard/filter/FilterSpec.scala:38]: here also

**Comment by danpersa** [comment]
lgtm, should I merge it?

**Comment by boopathi** [comment]
Yup :smiley:

**Pull request merged**